AjobK/Athena
# Back-End Folder Structure | Closes #55

Closes #55 

## What changed
- Added DB ERD 
- Added DB SQL dump

## How to test
1. Switch to branch and pull
2. Run XAMPP
3. Navigate to your PHPMyAdmin
4. Press import
5. Import the SQL DUMP file
6. Check for errors 
7. Check the ERD if you agree with the tables and columns, if not please suggest changes, if you'd like to see things added please suggest ideas etc.
